orc8r/tools/ansible/roles/docker/tasks/install_debian.yml

```diff
@@ -17,35 +17,38 @@
 
 - name: Install packages to allow apt to use a repository over HTTPS
   apt:
-    name: "{{ packages }}"
-    state: present
-    update_cache: true
-  vars:
-    packages:
+    name:
       - apt-transport-https
       - ca-certificates
       - curl
       - gnupg-agent
       - software-properties-common
       - python3-pip
+      - lsb-release
+    state: present
+    update_cache: true
 
 - name: Copy docker gpg key from codebase
   copy:
     src: docker.key
     dest: /tmp/docker.key
 
-- name: Add Docker’s official GPG key
+- name: Add Docker's official GPG key
   apt_key:
     file: /tmp/docker.key
     state: present
 
-- name: Add the stable repository for Docker
+- name: Add the focal repository for Docker
```

Comment:
Nit: Maybe it makes more sense to keep it as "stable" so that this doesn't need to be changed if the distribution is updated